### PR TITLE
Added multi-platform Orthanc server image

### PR DESCRIPTION
**Pull request by ccorley**

### CODE REVIEW

File: orthanc/orthanc.json [hunk 1]
```diff
@@ -0,0 +1,377 @@
+{
+  /**
```
Comment (ccorley) [inline orthanc/orthanc.json:2]: This is the proper format for the orthanc.json configuration file, despite the comments.

File: orthanc/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+COPY ./orthanc.json /etc/orthanc
+
+VOLUME [ "/var/lib/orthanc/db-v6" ]
+EXPOSE 4242
+EXPOSE 8042
+
+ENV TZ=America/New_York
+
+ENTRYPOINT [ "Orthanc" ]
+CMD [ "/etc/orthanc/" ]
```
Comment (dixonwhitmire) [inline orthanc/Dockerfile:34]: I was able to run the container in the LFH Connect compose stack, and validate that it works 🎉 !
I was curious about the CMD here. Do we need the trailing slash?

**Review by dixonwhitmire — APPROVED** [review]
I just had a minor question regarding the Dockerfile CMD

File: orthanc/Dockerfile (hunk 2 above)
Comment (ccorley) [reply orthanc/Dockerfile:34]: I believe we do, because the config filename is likely appended on Orthanc startup.

Comment (ccorley) [reply orthanc/Dockerfile:34]: I'll check this out and if it can be removed, I'll do it as a part of the next connect commit for an Orthanc route.

**Pull request merged**